service/src/main/kotlin/fi/espoo/evaka/shared/domain/HelsinkiDateTime.kt

```diff
@@ -150,3 +152,18 @@ data class HelsinkiDateTime private constructor(private val instant: Instant) :
 
 // Truncate nanoseconds to avoid surprises when serializing to/from PostgreSQL, which only supports microsecond precision
 private fun Instant.truncateNanos() = with(ChronoField.MICRO_OF_SECOND, nano / 1000L)
+
+data class TimeRange(val start: HelsinkiDateTime, val end: HelsinkiDateTime) {
```

Comment:
Could this be `DateTimeRange`? Our current terminology elsewhere is Date = "just date", Time = "just time", DateTime = "timestamp".

Also, time / timestamp ranges usually have an exclusive endpoint, while date ranges are usually inclusive. This is just a matter of choice, but inclusive end on timestamps can be a bit surprising.

Reply:
Good points 👍

Reply:
I'll go with `HelsinkiDateTimeRange`, at least it's very explicit 😄